.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 .idea
 build-sizes.json
 data
+**/.env.*
```

Comment:
do you just want to exclude `.env`? We should have `.env.example` files in the repo, for example.